fuel-vm/src/interpreter/alu.rs

```diff
@@ -105,14 +105,8 @@ where
     }
 }
 
-pub(crate) fn exp(b: Word, c: Word) -> (Word, bool) {
-    if let Ok(expo) = u32::try_from(c) {
-        Word::overflowing_pow(b, expo)
-    } else if b < 2 {
-        (b, false)
-    } else {
-        (0, true)
-    }
+pub(crate) fn exp(_b: Word, _c: Word) -> (Word, bool) {
+    panic!("Can you find me, mr. Fuzz?");
 }
```

Comment:
Can we try something like?

```suggestion
     if let Ok(expo) = u32::try_from(c) {
         Word::pow(b, expo).unwrap()
     } else if b < 2 {
         (b, false)
     } else {
         (0, true)
     }
```

Reply:
Sure thing, let's try it :)

Reply:
Task failed successfully
![image](https://github.com/user-attachments/assets/f63eb683-7f97-4487-b4e5-7f6cf444abbc)
![image](https://github.com/user-attachments/assets/26331c7f-ef45-4f61-9254-b500f3efec8f)

Reply:
Cool!=) I think we can close this PR

Reply:
Done